Userland/Libraries/LibUnicode/IDNA.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023, Simon Wanner <simon@skyrising.xyz>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <AK/Utf32View.h>
+#include <AK/Utf8View.h>
```

Comment:
`String.h` and `Utf8View.h` are unused in this header

Reply:
The header declares `ErrorOr<String> to_ascii(Utf8View domain_name, ...`

Reply:
Whoops I meant to delete this comment. At the point I left it, the header did not have that declaration. Ideally these headers would be added in the commit that adds that declaration, but I'll not be that nit picky here.